src/curp-command.proto

```diff
@@ -144,13 +184,11 @@ message PublishResponse {
 
 
 service Protocol {
-    rpc Propose (ProposeRequest) returns (ProposeResponse);
-    rpc WaitSynced (WaitSyncedRequest)
-            returns (WaitSyncedResponse);
-    rpc ProposeConfChange (ProposeConfChangeRequest) returns (ProposeConfChangeResponse);
+    rpc Propose(ProposeRequest) returns (ProposeResponse);
+    rpc WaitSynced(WaitSyncedRequest) returns (WaitSyncedResponse);
+    rpc ProposeConfChange(ProposeConfChangeRequest) returns (ProposeConfChangeResponse);
     rpc Publish (PublishRequest) returns (PublishResponse);
```

Comment:
Please unify the code style with other rpc services. It should be `rpc Publish(PublishRequest) returns (PublishResponse);`, without a space between `Publish` and `(PublishRequest)`

Reply:
Fixed in #23 [ref](https://github.com/xline-kv/curp-proto/pull/23/commits/c82a3663f82354397016bccb7fda55c0b10cf244)